content/07.pathogenesis.md

```diff
@@ -121,34 +125,36 @@ Coronaviruses such as SARS-CoV-1 are also able to evade the humoral immune respo
 As it becomes clear that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.
 
 ### Clinical Presentation of COVID-19
 
-Pathogenesis is closely linked with the clinical presentation of the disease caused by the virus.
-Reports have described diverse symptom profiles associated with COVID-19.
-Differences in the frequency of symptoms are found when comparing both between institutions in similar locations and between different regions.
+Pathogenesis is closely linked with the clinical presentation of the COVID-19 disease caused by this virus.
+Reports have described diverse symptom profiles associated with COVID-19, with a great deal of variability both within and between institutions and regions.
 A large study from Wuhan, China conducted early in the pandemic identified fever and cough as the two most common symptoms that patients reported at hospital admission [@doi:10.1056/NEJMoa2002032], while a retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
 This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, suggesting that the virus preferentially targets cells located in the lower respiratory tract.
-However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] revealed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
-For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients upon presentation, while another study [@doi:10.1056/NEJMc2010419] reported cough, fever, and dyspnea as the most common presenting symptoms.
+However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across individuals.
+For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in 20.4% of patients upon presentation, with fever being present in 30.7%, while another study [51] reported cough (79.4%), fever (77.1%), and dyspnea (56.5%) as the as the most common presenting symptoms; both of these studies considered only hospitalized patients.
+The variability of both which symptoms present and their severity has presented challenges for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.
```

Comment:
```suggestion
```
Thanks @agitter! This was removed in #796.
@ajlee21, I checked carefully and I am pretty confident that all of the corrections to duplications from #772 are fixed here! It also looks like a unique sentence was deleted and it's added back in below (line 142)